includes/admin.form.inc

```diff
@@ -34,6 +34,14 @@ function islandora_openseadragon_admin($form, &$form_state) {
     '#default_value' => $settings['djatokaServerBaseURL'],
     '#description' => t('The location of the Djatoka server OpenURL resolver.'),
   );
+  // Djatoka local file or PID test.
+  $form['islandora_openseadragon_settings']['djatokaJP2Image'] = array(
+    '#type' => 'textfield',
+    '#title' => t('JP2 Image Test'),
+    '#description' => t('Enter the PID of an object that has a JP2 datastream to test against Djatoka server. Defaults to LOC map image. Alternatively, you can use a local JP2 by entering this path: ' . drupal_get_path('module', 'islandora_openseadragon') . '/test/test.jp2'),
```

Comment:
Just relaying Travis's warning here.
`Don't use a variable in the first argument to t()`